config/samples/deployment.skyscanner.net_v1alpha1_progressiverollout.yaml

```diff
@@ -1,7 +1,33 @@
 apiVersion: deployment.skyscanner.net.deployment.skyscanner.net/v1alpha1
```

Comment:
is this a test that gets run?

Reply:
I think that I wrote the domain twice when initializing the project. Good catch.